.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+---
+CIP: ?
+Title: ?
+Authors: John Doe <john.doe@email.domain>
+Status: Draft
+Type: Core | Process | Informational
+Created: YYYY-MM-DD
+License: CC-BY-4.0
+---
+
+## Abstract <!-- A short (~200 word) description of the technical issue being addressed -->
```

Comment:
Should the abstract describe just the technical issue being addressed, the proposed solution, or both? I think probably both.

Reply:
Both indeed.